Code/GraphMol/DistGeomHelpers/Wrap/testDistGeom.py

```diff
@@ -375,16 +375,24 @@ def test8MultiThreadMultiConf(self):
             ee = ff.CalcEnergy()
             energies.append(ee)
 
-        mol = Chem.AddHs(Chem.MolFromSmiles("CC(C)(C)c(cc12)n[n]2C(=O)/C=C(N1)/COC"))
-        cids = rdDistGeom.EmbedMultipleConfs(mol,200,maxAttempts=30,randomSeed=100,numThreads=4)
+        mol2 = Chem.AddHs(Chem.MolFromSmiles("CC(C)(C)c(cc12)n[n]2C(=O)/C=C(N1)/COC"))
```

Comment:
These changes (and the ones in testDGeomHelpers.cpp) are reasonable, but really aren't part of this PR.
Please keep PRs single topic in the future.